BlogTemplate.Tests/Models/BlogDataStoreTests.cs

```diff
@@ -234,6 +234,49 @@ public void GetAllPosts_ReturnsList()
             Assert.NotEmpty(posts);
         }
 
+        [Fact]
+        public void FindComment_SwitchIsPublicValue()
+        {
+
+            BlogDataStore testDataStore = new BlogDataStore();
+            Post testPost = new Post
+            {
+                Slug = "Test-slug",
+                Title = "Test title",
+                Body = "Test body",
+                PubDate = DateTime.Now,
+                LastModified = DateTime.Now,
+                IsPublic = true,
+                Excerpt = "Test excerpt"
+            };
+            var comment1 = new Comment
+            {
+                AuthorName = "Test name",
+                AuthorEmail = "Test email",
+                Body = "test body",
+                PubDate = DateTime.Now,
+                IsPublic = true
+            };
+            var comment2 = new Comment
+            {
+                AuthorName = "Test name",
+                AuthorEmail = "Test email",
+                Body = "test body",
+                PubDate = DateTime.Now,
+                IsPublic = true
+            };
+            testPost.Comments.Add(comment1);
+            testPost.Comments.Add(comment2);
+            testDataStore.SavePost(testPost);
+            testDataStore.SaveComment(comment1, testPost);
```

Comment:
Isn't this redundant with SavePost saving the comments?  Actually, is this adding a 3rd/4th comment?